Mathlib/Data/ENat/Basic.lean

```diff
@@ -281,4 +281,21 @@ lemma one_le_iff_ne_zero_withTop {n : WithTop ℕ∞} :
   ⟨fun h ↦ (zero_lt_one.trans_le h).ne',
     fun h ↦ add_one_nat_le_withTop_of_lt (pos_iff_ne_zero.mpr h)⟩
 
+lemma add_one_pos : 0 < n + 1 := by
+  rw [← Order.one_le_iff_pos]
+  exact le_add_self
```

Comment:
```suggestion
lemma add_one_pos : 0 < n + 1 :=
  succ_def n ▸ Order.bot_lt_succ n
```
This change optional, I just think it is more clear to use `Order.bot_lt_succ` because it is basically the same lemma.